getting_started_modules.md

```diff
@@ -0,0 +1,256 @@
+## Getting Started
+
+### Content
+1. [Goal of This Guide](#goal-of-this-guide)
+2. [Directory Structure](#directory-structure)
+3. [The development process](#the-development-process)
+  1. [1. Building Docker Image](#1-building-docker-image)
+  2. [2. Deploying Container With Nomad](#2-deploying-container-with-nomad)
+    1. [a. Making image available to Nomad](#a-making-image-available-to-nomad)
+    2. [b. Creating a nomad job](#b-creating-a-nomad-job)
+  3. [3. Creating the Terraform Module](#3-creating-the-terraform-module)
+    1. [main.tf](#maintf)
+    2. [variables.tf](#variablestf)
+    3. [outputs.tf](#outputstf)
+  4. [4. Using a Terraform Module](#4-using-a-terraform-module)
+    1. [a. How To Use The Module and Run It Manually](#a-how-to-use-the-module-and-run-it-manually)
+    2. [b. Using Ansible To Run the Code in the Previous Step On Startup](#b-using-ansible-to-run-the-code-in-the-previous-step-on-startup)
+  5. [5. Making The Nomad Job More Dynamic With Terraform Variables](#5-making-the-nomad-job-more-dynamic-with-terraform-variables)
+  6. [6. Integrating The Nomad Job With Vault](#6-integrating-the-nomad-job-with-vault)
+  7. [7. CI/CD Pipeline To Continuously Test The Module When Changes Are Made](#7-cicd-pipeline-to-continuously-test-the-module-when-changes-are-made)
+
+
+### Goal of This Guide
+
+> :warning: Read the section `Description - what & why` in [README.md](/README.md) to get a quick introduction to what this repo is.
+
+The template-repo you found this README in is specifically built to make it as easy and quick as possible to make terraform modules, and then test them inside the [vagrant
+-hashistack](https://app.vagrantup.com/fredrikhgrelland/boxes/hashistack) box. 
+
+This guide aims to show you how to use this template. 
+The steps we are going to walk through are as follows:
+
+1. [Optional] [Building Docker Image](#1-building-docker-image)
+1. [Deploying Container With Nomad](#2-deploying-container-with-nomad)
+1. [Creating The Terraform Module](#3-creating-the-terraform-module)
+1. [Using a Terraform Module](#4-using-a-terraform-module)
+1. [Making The Nomad Job More Dynamic With Terraform Variables](#5-making-the-nomad-job-more-dynamic-with-terraform-variables)
+1. [Integrating The Nomad Job With Vault](#6-integrating-the-nomad-job-with-vault)
+1. [CI/CD Pipeline To Continuously Test The Module When Changes Are Made](#7-cicd-pipeline-to-continuously-test-the-module-when-changes-are-made)
```

Comment:
```suggestion
1. [Optional] [Building Docker Image](#1-building-docker-image)
2. [Deploying Container With Nomad](#2-deploying-container-with-nomad)
3. [Creating The Terraform Module](#3-creating-the-terraform-module)
4. [Using a Terraform Module](#4-using-a-terraform-module)
5. [Making The Nomad Job More Dynamic With Terraform Variables](#5-making-the-nomad-job-more-dynamic-with-terraform-variables)
6. [Integrating The Nomad Job With Vault](#6-integrating-the-nomad-job-with-vault)
7. [CI/CD Pipeline To Continuously Test The Module When Changes Are Made](#7-cicd-pipeline-to-continuously-test-the-module-when-changes-are-made)
```

Reply:
I believe just using 1s to signalise a ordered list is normal in markdown, but I guess I can change it anyways!